apps/easypid/src/crypto/WalletServiceProviderClient.ts

```diff
@@ -0,0 +1,52 @@
+import type { SecureEnvironment } from '@animo-id/expo-secure-environment'
+
+export class WalletServiceProviderClient implements SecureEnvironment {
```

Comment:
Also, i'm not really sure why this implements secure env? We're going yo have to interact a lot more with the wallet provider backend (for attestations etc). I think it makes more sense to create a separate service.

Also my understanding was we'd also create other keys in the HSM, but it now looks we only do the PID key?